lua/textcase/init.lua

```diff
@@ -35,6 +35,7 @@ local M = {
     toggle_boolean = presets.toggle_boolean,
   },
   setup = presets.setup,
+  init = presets.Initialize,
```

Comment:
I exposed it for the integration tests to work and then I adjusted it in start.vim as well. Does this make sense?

Reply:
Should be fine